app/src/components/features/mocksV2/MockEditor/Editor/index.tsx

```diff
@@ -119,6 +119,9 @@ const MockEditor: React.FC<Props> = ({
       headersDict = {};
     }
 
+    const cleanedEndpoint = cleanupEndpoint(endpoint);
+    setEndpoint(cleanedEndpoint);
+
     const tempMockData: MockEditorDataSchema = {
```

Comment:
rename to `mockData`, variable is not temporary, it is the final resulting value